products.js

```diff
@@ -39,31 +45,93 @@ function tunedImageFnodes(nodeToCssMap, coeffImgSize = 2.5, coeffImgHasSrc = 2.5
     }
 
     function imageTitle(fnode) {
-      if (title === undefined) {
-        return 1;
+      const attrs = ['title', 'alt'];
+      for(let i = 0; i < attrs.length; i++){
+        if(fnode.element.getAttribute(attrs[i]) &&
+            (fnode.element.getAttribute(attrs[i]).includes(title) || title.includes(fnode.element.getAttribute(attrs[i])))){
+              return 100 * coeffImgTitle;
```

Comment:
Will do! I also tried out another set of coefficients for prices that I got from the optimizer that give the same accuracy on the training data, and is giving 4% better accuracy, I could use this instead? (will be at 94%)